docs/getting-started-guides/kubeadm.md

```diff
@@ -98,7 +98,16 @@ For each host in turn:
 
 * SSH into the machine and become root if you are not already (for example,
   run `sudo su -`).
-* If the machine is running Ubuntu or HypriotOS, run:
+
+* If the machine is running Ubuntu or Debian:
+
+  ``` bash
+  apt install docker.io snapd
+  snap install kubeadm --classic
```

Comment:
that said, someone could target a specific release, like 1.6 with `snap install kubeadm --classic --channel 1.6/stable` where `stable` could be one of four: `stable`, `candidate`, `beta`, `edge`.